notebooks/image_models/solutions/serving.ipynb

```diff
@@ -0,0 +1,1639 @@
+{
```

Comment:
<p>Unmatched quote at the end of this sentence it seems</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/233/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
<p>Thanks! Removed.</p>